src/api.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const API_BASE_URL = 'http://localhost:3001';
```

Comment:
- I Think here it should be 3000 not 3001 because we are connecting the backend that runs on 3000 not on 3001 🔥💯